### DISCUSSION TITLE
Allow date input with and without leading zero

### DISCUSSION THREAD

**Discussion by FrEaKmAn** [uNmAnNeR/imaskjs#541] [Q&A]
I'm using Date format as showcased here https://imask.js.org/guide.html#masked-date

How could I enter date without leading zeros, so entering 5 and then . (dot) or some other delimiter? I would like to automatically convert to 05. , so user doesn't have to always write leading zero.

**Comment by FrEaKmAn** [comment]
Any feedback about this problem?

**Reply by nandapdtr** [reply]
I am also waiting for this feature

**Comment by uNmAnNeR** [comment]
use `autofix: 'pad'` since v6.3.0

**Reply by FrEaKmAn** [reply]
Yes, this is better but not perfect. I would still like to write (in case of DD.MM.YYYY format) 1 and then press dot to start inputting month.
